src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -36,10 +38,126 @@
+                                                            ))}]])}
+                      {:header-component (header "Name") :starting-width 200
+                       :cell-renderer (fn [row-num conf]
+                                        (cell [:a {:href "javascript:;"
+                                                   :style {:color (:button-blue style/colors)
+                                                           :textDecoration "none"}} (conf "name")]))}
+                      {:header-component (header "Namespace") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (conf "namespace")))}
+                      {:header-component (header "Type") :starting-width 100
+                       :cell-renderer (fn [row-num conf] (cell (conf "rootEntityType")))}
+                      {:header-component (header "Workspace Name") :starting-width 160
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "workspaceName") "namespace") ":"
+                                                                 ((conf "workspaceName") "name"))))}
+                      {:header-component (header "Method") :starting-width 210
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "methodStoreMethod") "methodNamespace") ":"
+                                                                 ((conf "methodStoreMethod") "methodName") ":"
+                                                                 ((conf "methodStoreMethod") "methodVersion"))))}
+                      {:header-component (header "Config") :starting-width 290
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "methodStoreConfig") "methodConfigNamespace") ":"
+                                                                 ((conf "methodStoreConfig") "methodConfigName") ":"
+                                                                 ((conf "methodStoreConfig") "methodConfigVersion"))))}
+                      {:header-component (header "Inputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "inputs"))))}
+                      {:header-component (header "Outputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "outputs"))))}
+                      {:header-component (header "Prerequisites") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "prerequisites"))))}]
+            :data (:method-confs props)
+            :row-props (fn [row-num conf]
+                         {:style {:fontSize "80%" :fontWeight 500
+                                  :paddingTop 10 :paddingBottom 7
+                                  :backgroundColor (if (even? row-num) (:background-gray style/colors) "#fff")}})})])])})
+
+
+(defn- modal-import-background [state]
```

Comment:
Matt, I wonder if it's the re-usage of the code that motivated your idea for the generalization?  By "pull out the common styling elements", do you mean to move them to a parent div so that the styles "cascade" down?